unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs

```diff
@@ -123,11 +123,8 @@ internal string GetUrl(string endPoint, params (string paramName, string paramVa
             }
         }
 
-        internal string GetLambdasUrl()
-        {
-            // TODO (Santi): This should use catalyst.lambdasUrl instead the hardcode string
-            return "https://peer.decentraland.org/lambdas/";
-        }
+        internal string GetLambdasUrl() =>
+            "https://peer.decentraland.org/lambdas/";
```

Comment:
should this string be based on the domain used e.g. `https://peer.decentraland.zone/lambdas/`

Reply:
This lambdas service will be used ONLY for `org` scenarios. For the rest ones we will keep using the current kernel's flow.